src/components/validator/StakeCard.tsx

```diff
@@ -4,12 +4,12 @@ import { useState } from "react";
 import {
     LargeWhiteButton,
     RoundButton,
-    ErrorMessage,
     LargeInput,
-    formatBalance
+    formatBalance,
+    inputToKens
 } from "../common/core";
 import { Card, CardHeader, CardTitle, CardBody } from "../common/card";
-
+import BigNumber from "bignumber.js";
```

Comment:
Do we use this library in this file? I dont see it

Reply:
No. It was omitted after a small refactor.